nilrt-base-system-image.postinst: Enable persistent logging

pratheekshasn opened the pull request:
### Summary of Changes

To include logs from /var/log in the technical report created by Hardware Config Utility, PersistentLogs needs to be enabled since that path is mounted on tmpfs.


### Justification

[AB#3110389](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/3110389)


### Testing

Tested that the new installed base system image has PersistentLogs enabled after installation.

| Before | After |
|-----|-----|
| ![image](https://github.com/user-attachments/assets/56a3d24b-7212-4d80-8a14-40833125259a) | ![image](https://github.com/user-attachments/assets/7b508718-0bff-458d-8bf9-8ebaa065a862) |



* [x] I have built the core package feed with this PR in place. (`bitbake packagefeed-ni-core`)


### Procedure

* [x] I certify that the contents of this pull request complies with the [Developer Certificate of Origin](https://github.com/ni/nilrt/blob/HEAD/docs/CONTRIBUTING.md#developer-certificate-of-origin-dco).

chaitu236 commented on recipes-support/lvrt-logging/files/logging_paths.ini:
```diff
@@ -0,0 +1,7 @@
+[Paths]
```
You have these changes in https://github.com/ni/meta-nilrt/pull/839. Why duplicate?

pratheekshasn replied:
This was a branching mistake, I have fixed this now.

chaitu236 commented on recipes-core/images/nilrt-container-base.bb:
```diff
@@ -0,0 +1,22 @@
+SUMMARY = "Custom Docker-compatible image"
```
This file does not belong in this PR

pratheekshasn replied:
This was a branching mistake, I have fixed this now.

chaitu236 reviewed (changes requested):
I see no changes to enable logging in this PR.

If this is still a draft PR, please mark it as such

pratheekshasn commented:
The first commit has the changes to enable `PersistentLogs`. It has somehow applied an older commit on top of the new commit, I must have made a branching mistake. I'll fix this.

chaitu236 commented on recipes-core/images/files/nilrt-base-system-image.postinst:
```diff
@@ -17,6 +17,7 @@ grub-editenv /boot/grub/grubenv unset enable_initramfs
 
 # Enable ssh by default when installing SystemLink.
 /usr/local/natinst/bin/nirtcfg -s section=systemsettings,token=sshd.enabled,value=True
+/usr/local/natinst/bin/nirtcfg -s section=systemsettings,token=PersistentLogs.enabled,value=True
```
Add a new line before this to separate from ssh, and also add a comment

pratheekshasn replied:
I have added a comment now.

pratheekshasn committed:
nilrt-base-system-image.postinst: Enable PersistentLogs

To include logs from /var/log in the technical report created by
Hardware Config Utility, PersistentLogs needs to be enabled since
that path is mounted on tmpfs.

Signed-off-by: pratheekshasn <pratheeksha.s.n@ni.com>
